README.rst to README.md (#24067)
matplotlib/matplotlib: README.md

[![AppVeyor](https://ci.appveyor.com/api/projects/status/github/matplotlib/matplotlib?branch=main&svg=true)](https://ci.appveyor.com/project/matplotlib/matplotlib)
[![Codecov](https://codecov.io/github/matplotlib/matplotlib/badge.svg?branch=main&service=github)](https://codecov.io/github/matplotlib/matplotlib?branch=main)
[![LGTM](https://img.shields.io/lgtm/grade/python/github/matplotlib/matplotlib.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/matplotlib/matplotlib)

![image](https://matplotlib.org/_static/logo2.svg)

Matplotlib is a comprehensive library for creating static, animated, and
interactive visualizations in Python.

Check out our [home page](https://matplotlib.org/) for more information.

![image](https://matplotlib.org/_static/readme_preview.png)

Matplotlib produces publication-quality figures in a variety of hardcopy
formats and interactive environments across platforms. Matplotlib can be
used in Python scripts, Python/IPython shells, web application servers,
and various graphical user interface toolkits.

> ## **Install** 

Reviewer: ```suggestion
## Install
```
I don't think we want bold formatting on top of the heading. Also applies  to all following headings.

See the [install
documentation](https://matplotlib.org/stable/users/installing/index.html),
which is generated from `/doc/users/installing/index.rst`

## **Contribute**

> You\'ve discovered a bug or something else you want to change -

Reviewer: ```suggestion
You've discovered a bug or something else you want to change -
```
Escaping not needed in Markdown.

excellent!

> You\'ve worked out a way to fix it -- even better!

Reviewer: ```suggestion
You've worked out a way to fix it -- even better!
```
